services/worker/pyproject.toml

```diff
@@ -31,7 +31,7 @@ pypdf2 = "^3.0.1"
 python = "3.9.15"
 rarfile = "^4.0"
 scikit-learn = "^1.2.1"
-tensorflow-cpu = {version = "^2.11.0", markers = "sys_platform != 'darwin' or platform_machine != 'arm64'"}
+tensorflow = {version = "^2.11.0", markers = "sys_platform != 'darwin' or platform_machine != 'arm64'"}
```

Comment:
The docker images will include 2GB of useless binaries (for GPU) if we change back to tensorflow. See https://github.com/huggingface/datasets-server/pull/680/files.

I'm not sure what would be the best compromise to be able to work on MacOS locally and to have the smallest docker images possible.

Reply:
hmm let me try alternatives then